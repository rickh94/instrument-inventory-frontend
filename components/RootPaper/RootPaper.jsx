import { styled } from '@material-ui/styles'
import { Paper } from '@material-ui/core'

import { root } from '../../globalStyles'

export default styled(Paper)({
  ...root,
})
