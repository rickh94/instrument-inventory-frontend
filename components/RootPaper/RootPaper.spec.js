import React from 'react'
import RootPaper from './RootPaper'
import { render } from '@testing-library/react'

describe('<RootPaper />', () => {
  it('matches snapshot', () => {
    const { container } = render(<RootPaper />)
    expect(container).toMatchSnapshot()
  })
})
