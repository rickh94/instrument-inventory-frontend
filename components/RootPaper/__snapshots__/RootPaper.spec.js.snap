// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RootPaper /> matches snapshot 1`] = `
<div>
  <div
    class="MuiPaper-root MuiPaper-elevation1 MuiPaper-rounded WithStyles(ForwardRef(Paper))-root-1"
  />
</div>
`;
